packages/angular/module-federation/index.ts

```diff
@@ -1,2 +0,0 @@
-export { withModuleFederation } from '../src/utils/mf/with-module-federation';
```

Comment:
Should do still support the exports, and add deprecation notice for v22?

It's possible that the migration doesn't work 100%.